tests/quantization/test_waveq_loss.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from nncf.quantization.waveq_loss import WaveQLoss, get_quant_module_params
+from nncf.quantization.algo import QuantizationController
+import torch
+import math
+from torch.utils import tensorboard as tb
+import matplotlib.pyplot as plt
+from tests.test_helpers import BasicConvTestModel, create_compressed_model_and_algo_for_test
+from tests.quantization.test_algo_quantization import get_basic_quantization_config, \
+    get_basic_asym_quantization_config, OnesDatasetMock
+from nncf.initialization import register_default_init_args
+
+kernel_size = 10
```

Comment:
avoid using of global variables and definitions - it's [evil](https://stackoverflow.com/questions/19158339/why-are-global-variables-evil) `pytest.fixture` is more suitable. there're many examples in repo.